README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $(GOPATH)/github.com/opencontainers/runc
```

Comment:
Yes, but other go get packages are generally under the src directory hence just to be consistent :)